mkosi

```diff
@@ -1438,6 +1438,17 @@ def install_extra_trees(args, workspace, for_cache):
         for d in args.extra_trees:
             enumerate_and_copy(d, os.path.join(workspace, "root"))
 
+def install_skeleton_trees(args, workspace, for_cache):
+    if args.skeleton_trees is None:
+        return
+
+    if for_cache:
+        return
```

Comment:
why this bit? I think this should be included in the cache files really, otherwise the skeleton tree would not be taken into account when we invoke mkosi in incremental mode, but it really should be in that case, so that we can properly cache the result of the first dnf/apt-get run

Reply:
I misread what for_cache was and just copied what was there for extra_tree.